mdrmsdplot.py

```diff
@@ -38,15 +38,15 @@
 n = u1.trajectory.n_frames
 
 div = args.step
-rmat = np.zeros((n/div,n/div))
+rmat = np.zeros((int(n/div),int(n/div)))
```

Comment:
use // (double slash) to indicate integer division - it is cleaner.

Reply:
Fixed, I figured there would be a cleaner way. Thanks for the tip!